ts/Core/Templating.ts

```diff
@@ -185,9 +194,9 @@ function format(str = '', ctx: any, chart?: Chart): string {
         matches = [],
         floatRegex = /f$/,
         decRegex = /\.(\d)/,
-        lang = chart?.options.lang || defaultOptions.lang,
-        time = chart && chart.time || defaultTime,
-        numberFormatter = chart && chart.numberFormatter || numberFormat;
+        lang = templateOwner?.options?.lang || defaultOptions.lang,
+        timeInstance = templateOwner?.time || defaultTime,
```

Comment:
We can call the instance just `time` like we do everywhere else
```suggestion
        time = templateOwner?.time || defaultTime,
```